### PR TITLE
add a make task for generating dev totp creds

### PR DESCRIPTION
**Pull request by ewdurbin**

### CODE REVIEW

File: docs/dev/development/getting-started.rst [hunk 1]
```diff
@@ -321,7 +321,25 @@ the string ``password``. You can log in as any account at
 http://localhost:80/account/login/.
 
 To log in as an admin user, log in as ``ewdurbin`` with the password
-``password``. Due to session invalidation, you may have to login twice.
+``password``. The users below all have TOTP 2FA enabled with a known
+secret. You can generate a TOTP value for logging in using:
+
+.. code-block:: console
+
+    $ make totp
```
Comment (miketheman) [inline docs/dev/development/getting-started.rst:329]: Excellent. Should this move lower, and either add or replace other totp-cli commands I added yesterday?

Comment (ewdurbin) [reply docs/dev/development/getting-started.rst:329]: I don't have too much of an opinion here. I do think the totp-cli bits could be helpful for people who want to add TOTP to other random accounts.

**Review by miketheman — APPROVED** [review]

**Review by woodruffw — APPROVED** [review]
Bless you for this

File: docs/dev/development/getting-started.rst (hunk 1 above)
Comment (ewdurbin) [reply docs/dev/development/getting-started.rst:328]: moved things around a bit in https://github.com/pypi/warehouse/pull/16263/commits/be64a47311ffa3b9274e4fe1196e75ed26cc3b5f

**Pull request merged**